lib/open_rtb_ecto/v2/native/helper.ex

```diff
@@ -0,0 +1,23 @@
+defmodule OpenRtbEcto.V2.Native.Helper do
+  import Ecto.Changeset
+
+  # Each object may contain only one of title, img, data or video.
+  def validate_media(asset) do
```

Comment:
Let's call this arg `changeset` so it is clear that is what it takes. Maybe we should add a typespec too?
```suggestion
  def validate_media(changeset) do
```